frontend/src/views/SecretMainPage/components/SecretListView/SecretItem.tsx

```diff
@@ -115,6 +115,9 @@ export const SecretItem = memo(
     const hasComment = Boolean(watch("comment"));
     const hasReminder = Boolean(watch("reminderRepeatDays"));
 
+    const secretReminderRepeatDays = watch("reminderRepeatDays");
```

Comment:
Your already watching for the same prop at top. Can you change this to one watch please